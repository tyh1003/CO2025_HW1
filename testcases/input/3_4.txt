12
65 56 1 34 84 45 92 88 79 62 58 28
