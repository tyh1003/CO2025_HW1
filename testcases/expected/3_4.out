1 28 34 45 56 58 62 65 79 84 88 92 
